javascript/lib/api/src/model/things.model.ts

```diff
@@ -60,8 +60,8 @@ export class Thing extends EntityWithId {
   }
 
   public toObject(): object {
-    const featuresObj = Features.toObject(this.features);
-    const aclObj = Acl.toObject(this._acl);
+    const featuresObj = this.features ? Features.toObject(this.features) : undefined;
```

Comment:
On master, it's
```
const featuresObj = Features.toObject(this.features);
const aclObj = Acl.toObject(this._acl);
```
Do we need this extra check?

Reply:
No, I think i merged this wrong